ncm-dirperm/src/main/pan/components/dirperm/schema.pan

```diff
@@ -30,11 +30,13 @@ function dirperm_permissions_valid = {
 };
 
 type structure_dirperm_entry = {
-    'path'    : string with match(SELF, '^/')
-    'perm'    : string with match(SELF,'[0-7]{3,4}')
-    'owner'   : string with match(SELF, '\w+(:\w+)?')
-    'type'    : string with match(SELF, 'd|f')
-    'initdir' ? string[]
+    'path'       : string with match(SELF, '^/')
+    'perm'       : string with match(SELF,'[0-7]{3,4}')
+    'owner'      : string with match(SELF, '\w+(:\w+)?')
+    'type'       : string with match(SELF, 'd|f')
+    'initdir'    ? string[]
+    'checkmount' : boolean = false
+    'notroot'    : boolean = false
```

Comment:
i am not a big fan of negated option names? can we invert the meaning of the name (and logic of the code)?
so here
```pan
'root' : boolean = true
```